glacier/src/actions/add-field.ts

```diff
@@ -0,0 +1,8 @@
+import {ReduxStandardAction} from "./";
+import {Field} from "../model";
+
+export type AddFieldAction = ReduxStandardAction<"ADD_FIELD", Field[]>;
+
+export function createAddFieldAction(fields: Field[]): AddFieldAction {
```

Comment:
Just a point of clarifying the language used: If the function is named "AddField", it should accept a single field. If it can add multiple fields, it should be called "AddFields". Choose one.